ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.12.yaml

```diff
@@ -33,13 +33,13 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: installer-rehearse-aws
+- as: installer-rehearse-aws-mtu-migrate
```

Comment:
Here do not need to change the job name, suggest to keep it unchanged.

Reply:
Good idea. Thanks. Taken care of

Reply:
Once that, https://github.com/openshift/release/pull/32171/files#diff-99e029f94e9d5b7bbffc17380fbcf9794b70d267a2e9144171e2aac459a664b4 should have no change, forget to `make registry-metadata` in your local?

Reply:
sorry, yep. Just pushed `make update`